packages/database/drizzle/0064_living_iron_monger.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE "user_performance" ADD COLUMN "dao_acct" varchar(44);--> statement-breakpoint
+ALTER TABLE "user_performance" ADD COLUMN "total_volume" numeric(40, 20) DEFAULT '0.0' NOT NULL;--> statement-breakpoint
+ALTER TABLE "user_performance" ADD COLUMN "tokens_bought_resolving_market" numeric(40, 20) DEFAULT '0.0' NOT NULL;--> statement-breakpoint
+ALTER TABLE "user_performance" ADD COLUMN "tokens_sold_resolving_market" numeric(40, 20) DEFAULT '0.0' NOT NULL;--> statement-breakpoint
+ALTER TABLE "user_performance" ADD COLUMN "volume_bought_resolving_market" numeric(40, 20) DEFAULT '0.0' NOT NULL;--> statement-breakpoint
+ALTER TABLE "user_performance" ADD COLUMN "volume_sold_resolving_market" numeric(40, 20) DEFAULT '0.0' NOT NULL;--> statement-breakpoint
+ALTER TABLE "user_performance" ADD COLUMN "buy_orders_count" bigint DEFAULT 0 NOT NULL;--> statement-breakpoint
+ALTER TABLE "user_performance" ADD COLUMN "sell_orders_count" bigint DEFAULT 0 NOT NULL;--> statement-breakpoint
+-- ALTER TABLE "user_performance" ADD COLUMN "updated_at" timestamp with time zone DEFAULT now() NOT NULL;--> statement-breakpoint
```

Comment:
We're at about that time to version the db and just start fresh the migrations.

Reply:
yahh